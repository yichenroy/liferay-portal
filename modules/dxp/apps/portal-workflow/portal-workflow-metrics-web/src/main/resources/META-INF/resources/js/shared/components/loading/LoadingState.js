import React from 'react';

/**
 * LoadingState.
 * @extends React.Component
 */
export default class LoadingState extends React.Component {
	render() {
		return <span aria-hidden='true' className='loading-animation' />;
	}
}
