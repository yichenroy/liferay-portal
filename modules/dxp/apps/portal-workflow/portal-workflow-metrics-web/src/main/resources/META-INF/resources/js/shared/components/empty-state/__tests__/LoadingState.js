import LoadingState from '../LoadingState';
import React from 'react';

test('Should test component render', () => {
	const component = shallow(<LoadingState />);

	expect(component).toMatchSnapshot();
});