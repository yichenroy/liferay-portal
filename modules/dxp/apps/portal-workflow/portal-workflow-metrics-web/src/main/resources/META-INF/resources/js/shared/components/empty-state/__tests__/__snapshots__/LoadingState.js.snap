// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should test component render 1`] = `
<span
  aria-hidden="true"
  className="loading-animation"
/>
`;
